specification/db_migration.md

```diff
@@ -1,3 +1,13 @@
+# 0.8.3
```

Comment:
@alankm , I think you missed to document the new table "instance_messages"?

Reply:
Intentionally, but please do sanity check me and let me know if it's a problem. I don't believe those docs need to cover every change, they only need to cover the upgrade strategy. Since our db initialization includes `CREATE TABLE IF NOT EXISTS`, I expect this table to be created properly with no human intervention. Do you agree?

Reply:
Just wanted to make sure that you didn't miss it.